package.json

```
@@ -55,7 +55,7 @@
     "semver": "^7.3.5",
     "sirv": "^1.0.17",
     "ts-jest": "^27.0.7",
-    "ts-node": "^10.1.0",
+    "ts-node": "^10.1.0", 
```

this file doesn't need to be touched
```suggestion
    "ts-node": "^10.1.0",
```